### PR TITLE
Clean up repo and add README

### PR DESCRIPTION
**Pull request by proudmuslim-dev**
You're lucky I don't know TS or I would be roasting your code rn

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.vscode
+.DS_Store
+.lock
+
+# Node modules - npm
+node_modules/
```
Comment (TymanWasTaken) [inline .gitignore:6]: You do realize this is deno where node_modules isn't a thing right

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,7 +0,0 @@
-{
```
Comment (TymanWasTaken) [inline .vscode/settings.json:1]: Why delet

**Review by TymanWasTaken — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (proudmuslim-dev) [reply .gitignore:6]: It's templated

Comment (proudmuslim-dev) [reply .gitignore:6]: That can be changed later

File: .vscode/settings.json (hunk 2 above)
Comment (proudmuslim-dev) [reply .vscode/settings.json:1]: Because .vscode shouldn't be in the repo

Comment (TrashCan69420) [reply .vscode/settings.json:1]: @proudmuslim-dev it makes it so deno actually works in vsc nerd

**Review by TrashCan69420 — COMMENTED** [review]
bad

**Comment by TrashCan69420** [comment]
> discord-deno
its discordeno

**Comment by proudmuslim-dev** [comment]
@TrashCan69420 Alright then let me fix the vscode thing

**Comment by proudmuslim-dev** [comment]
> > discord-deno
> > its discordeno

Yeah but I was copying the way the main lib put it

**Comment by proudmuslim-dev** [comment]
@TrashCan69420 It's done

**Comment by TrashCan69420** [comment]
gaming

**Comment by proudmuslim-dev** [comment]
Squashed

**Pull request merged**